DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: IRkernel.testthat
 Title: What the Package Does (one line, title case)
-Version: 0.2.0
+Version: 0.2.1
```

Comment:
That's weird you bump the version in the middle of the PR. With JS package, a commit is added on top of master and receive the appropriate tag. No idea if there is any similar practice for R though.

Reply:
I didn't know but makes sense! Thank you, next time I'll do it the correct way.